core/Cargo.toml

```diff
@@ -31,8 +31,9 @@ std = [
     "tracing/std",
     "impl-serde/std",
     "primitive-types/std",
+    "polkadot-sdk/std",
 ]
-substrate-compat = ["sp-core", "sp-runtime"]
+substrate-compat = ["polkadot-sdk/sp-core", "polkadot-sdk/sp-runtime", "polkadot-sdk/std"]
```

Comment:
So sp-core wasn't std before and sp-runtime was I think.. I guess std is needed here for it to work?

Reply:
I wonder how many more deps it ends up bringing in; maybe worth checking?

Reply:
It should only affect `--feature --substrate-compat` and not the std or no-std which I'm okay with...

Sure, lemme check

Reply:
this branch brings 351 depends vs 350 on master, I think it's fine :)

Reply:
Yeah that's all good then!